subiquity/controllers/welcome.py

```diff
@@ -25,7 +25,11 @@
 
 class WelcomeController(SubiquityController):
 
-    model_name = "locale"
+    autoinstall_key = model_name = "locale"
+    autoinstall_default = 'en_US.UTF-8'
```

Comment:
C.UTF-8  is the default in cloud-images;

Reply:
Hmm. subiquity is more targeted towards humans, but maybe not for autoinstall cases. Easy to change later in any case.